<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreUI</class>
 <widget class="QWidget" name="ScoreUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Score Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel_2">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="idLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="matchLabel">
        <property name="text">
         <string>Match:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="matchComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="score1Label">
        <property name="text">
         <string>Score 1:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="score1SpinBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="score2Label">
        <property name="text">
         <string>Score 2:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="score2SpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="scoresTableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addScoreButton">
       <property name="text">
        <string>Add Score</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteScoreButton">
       <property name="text">
        <string>Delete Score</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateScoreButton">
       <property name="text">
        <string>Update Score</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>